x/wasm/keeper/recurse_test.go

```diff
@@ -91,15 +91,15 @@ func TestGasCostOnQuery(t *testing.T) {
 				Depth: 1,
 				Work:  50,
 			},
-			expectedGas: 2*GasWork50 + GasReturnHashed,
+			expectedGas: 2*GasWork50 + GasReturnHashed + 1,
```

Comment:
Verifying this update

Reply:
do you mind shedding light as to why we expect 1 extra gas here?

Reply:
Fyi @philipsu522 @udpatil this feels like it may be unrelated to this PR and possible rounding issues. Check this comment from before: https://github.com/sei-protocol/sei-wasmd/blob/UpdateLimitSimulationGasDecorator/x/wasm/keeper/recurse_test.go#L248